frontend/src/components/user/Details.tsx

```diff
@@ -63,6 +65,9 @@ const UserDetails: FunctionComponent<Props> = ({ logins }) => {
         {loginSection}
 
         <div className={styles.actions}>
+          <Link href='/wallet' passHref>
+            <Button type='primary'>View Wallet</Button>
```

Comment:
Missing translation (yeah, I know we probably want to move it, the translation still seems useful?)